celements-spring-security/src/main/java/com/celements/spring/security/OAuth2BeansConfig.java

```diff
@@ -0,0 +1,73 @@
+package com.celements.spring.security;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManagerResolver;
+import org.springframework.security.oauth2.client.InMemoryOAuth2AuthorizedClientService;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientProvider;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientProviderBuilder;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientService;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.security.oauth2.client.web.AuthenticatedPrincipalOAuth2AuthorizedClientRepository;
+import org.springframework.security.oauth2.client.web.DefaultOAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.web.OAuth2AuthorizedClientRepository;
+import org.xwiki.context.Execution;
+
+import com.celements.execution.XWikiExecutionProp;
+import com.xpn.xwiki.XWikiConstant;
+
+@Configuration
+public class OAuth2BeansConfig {
```

Comment:
```suggestion
public class OAuth2Config {
```
“Beans” is redundant. Every `@Configuration` defines beans.